Orange/data/table.py

```diff
@@ -1286,6 +1075,7 @@ def _get_matrix(M, cachedM, col):
 
         return distributions
 
+    # TODO: move this to contingency.py and use pandas instead if this code
```

Comment:
Move what where? We don't need more dead code. :S
Just replace the public interfaces with appropriate pandas' describe() and pivot_table() calls.